featurex/stimuli/video.py

```diff
@@ -1,8 +1,9 @@
-from featurex.stimuli import Stim, DynamicStim
+from __future__ import division
+from featurex.stimuli import DynamicStim
 from featurex.stimuli.image import ImageStim
 from featurex.core import Timeline, Event
-import six
 from moviepy.video.io.VideoFileClip import VideoFileClip
+import pandas as pd
 
 
 class VideoFrameStim(ImageStim):
```

Comment:
I think instead of adding a new `VideoKeyframeStim` class just to add a non-fps-based duration, we can allow the `VideoFrameStim` initializer to take an optional `duration` argument that defaults to `None`. Then if `duration=None`, we set to 1 / fps, just like the current approach, otherwise we use whatever was passed in as the value.